recipes/autoclick/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "autoclick" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autoclick-{{ version }}.tar.gz
+  sha256: 583b1c7e57eedd4a9a61b35ff5fbf491d3eb1fedda5895ffaffad7baaf6786d8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >3.6,<4.0
+    - poetry >=0.12
+    - poetry-core
+    - pip
+  run:
+    - python >3.6,<4.0
+    - click >7.0,<8.0
```

pyproject.toml says >=7 but setup.py says ==7 :-/

Let's try this for now.

```suggestion
    - click >=7.0,<8.0
```